source/process/training.rst

```diff
@@ -454,8 +454,8 @@ Meeting Agenda:
 - **Ice-breaker** - see Meeting Elements > Ice-breaker below for examples
 - **Roadmap check-in** - Review of roadmap status in current and next release
 - **Team updates** - Each development team gives a short update on their current top priorities
-- **Demos (optional)** - Team members show highlights of what's been completed this week. Relevant follow-ups noted
-- **Blind spots, Inspiration, Knowledge Share** - Colleagues share areas of concern and ask questions
+- **Demos** - Team members show highlights of what's been completed this week. Relevant follow-ups noted
+- **Blind spots, Inspiration, Knowledge Share** - Colleagues share areas of concern and ask questions. Items for discussion can be queued here.
```

Comment:
```suggestion
- **Blind spots, Inspiration, Knowledge Share** - Colleagues share areas of concern and ask questions. Proposals can be queued here.
```

Given the size of the meeting, discussions may not be the best use of time because it may take a while. I'd recommend having discussions outside of the meeting, and bringing proposals to the meeting